data/sql/updates/pending_db_world/rev_1667732623534207400.sql

```diff
@@ -0,0 +1,22 @@
+-- Fire brigade esES / esMX
+DELETE FROM `quest_offer_reward_locale` WHERE `ID` in (11360, 11439, 11440) AND `locale` in ('esES', 'esMX');
```

Comment:
SQL statements in uppercase please.  It will work that way, but it's not good style.

```suggestion
DELETE FROM `quest_offer_reward_locale` WHERE `ID` IN (11360, 11439, 11440) AND `locale` IN ('esES', 'esMX');
```